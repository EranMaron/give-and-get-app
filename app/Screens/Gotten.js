import React, { Component } from 'react'
import { Text, View } from 'react-native'

class Gotten extends Component {
    
    static navigationOptions = {
        title: 'Gets',
        headerStyle: {
        backgroundColor: 'purple',
        },
        headerTintColor: '#fff',
        headerTitleStyle: {
          fontWeight: 'bold',
        },
        tabBarOptions: {
          inactiveBackgroundColor: '#2a2a2a',  
          activeBackgroundColor: "purple",
          activeTintColor: "#fff",
          labelStyle: {
            fontSize: 15,
            paddingBottom: 10,
          },
        }
    }
    
  render() {
    return (
      <View>
        <Text> Gotten Screen </Text>
      </View>
    )
  }
}

export default Gotten
