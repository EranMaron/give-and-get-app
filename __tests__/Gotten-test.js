import 'react-native';
import React from 'react';
import Gotten from '../app/Screens/Gotten'
import renderer from 'react-test-renderer';

describe('Gotten functionality', () => {
  test('Gotten component renders correctly', () => {
    const snapshot = renderer.create(<Gotten/>).toJSON();
    expect(snapshot).toMatchSnapshot();
  });
})