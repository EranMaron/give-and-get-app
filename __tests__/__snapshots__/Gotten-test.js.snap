// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Gotten functionality Gotten component renders correctly 1`] = `
<View>
  <Text>
     Gotten Screen 
  </Text>
</View>
`;
